src/App.tsx

```diff
@@ -40,11 +41,11 @@ const App = () => {
         name: user.name,
         date: moment(user.created_at).format("DD MMMM YYYY"),
         login: user.login,
-        avatar: user.avatar_url
+        avatar_url: user.avatar_url
```

Comment:
please change this to `avatarUrl`. This project is following the camelCase convention for variable names not the snake_case.

Reply:
Acknowledged but won't it better to use `snake_case` for properties name?

Reply:
- Javascript follow the `camelCase` variable naming convention.
- Python follows `snake_case` variable naming convention.

Its better to use camelCase everywhere.